modules/terminate-agent-hook/variables.tf

```diff
@@ -39,36 +39,12 @@ variable "cloudwatch_logging_retention_in_days" {
   default     = 30
 }
 
-variable "lifecycle_heartbeat_timeout" {
-  description = "The amount of time, in seconds, for the instances to remain in wait state."
-  type        = number
-  default     = 90
-}
-
 variable "name_iam_objects" {
   description = "The name to use for IAM resources - roles and policies."
   type        = string
   default     = ""
 }
 
-variable "lambda_memory_size" {
```

Comment:
Can you motivate why dropping the configuration. I assume since the lambda is mainted by the module.

Reply:
Yeap. That's the point. We manage the code. So we should know the exact memory size. I think it's error prone to let the user decide it.

By the way: Not sure if it the memory size has been analyzed. The more memory you reserve the more CPUs you get and in turn the function might run faster. We should evaluate that.

Reply:
#631 tackles this